### DISCUSSION TITLE
MacOs

### DISCUSSION THREAD

**Discussion by alissonrodrigo** [viniciussanchez/RESTRequest4Delphi#112] [General]
Estou fazendo uma requisição HTTPS e no Windows e Android está vindo a resposta normalmente mas quando compilo para MacOs a resposta chega em branco, alguém já passou por isso?

**Comment by PedroAlbuquerqueQueiroz** [comment]
Se o retorno for OK, mas apenas o conteudo não seja possivel de ser carregado ou vazio, no content, pode ser que voce nao esteja conseguindo disserializar o response.
